frontend/src/components/managePermissions/ManagePermissions.vue

```diff
@@ -128,6 +129,26 @@ const deleteAppAdmin = async (admin: FamAppAdminGetResponse) => {
         );
     }
 };
+
+const deleteDelegatedAdminAssignment = async (
+    delegatedAdminAssignment: FamAccessControlPrivilegeGetResponse
+) => {
+    try {
+        delegatedAdmins.value = await deleteAndRefreshDelegatedAdmin(
+            delegatedAdminAssignment.access_control_privilege_id
+        );
+
+        setNotificationMsg(
+            Severity.Success,
+            `You removed ${delegatedAdminAssignment.role.role_name} access to ${delegatedAdminAssignment.user.user_name}`
```

Comment:
I wonder if we should say "access" or "privilege"... this is like removing the privilege to manage that role

Reply:
yeah, it make sense saying privilege instead of access

Reply:
Hi Jota, let's change the access to privilege for now, `removed role_name from user_name`, also let's update the `deleteUserRoleAssignment` method, I feel it should be remove from, not remove to?

Reply:
ok, I'll make the change! Thanks!